src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/CreateDescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,503 @@
+                      "validationErrors": {
+                        "$.codeValue": [
+                          "codeValue is required."
+                        ]
+                      },
+                      "errors": []
+                    }
+                  """
+
+        # Ignored because we do not have namespace security for descriptors yet. DMS-81
+        @ignore
+        Scenario: 04 Post a Descriptor using an invalid namespace
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "xxxx",
+                        "description": "Wrong Value",
+                        "namespace": "uri://.org/wrong",
+                        "shortDescription": "Wrong Value"
+                    }
+                  """
+             Then it should respond with 403
+              And the response body is
+                  """
+                    {
+                        "detail": "Access to the resource could not be authorized. The 'Namespace' value of the resource does not start with any of the caller's associated namespace prefixes ('uri://ed-fi.org', 'uri://gbisd.org', 'uri://tpdm.ed-fi.org').",
+                        "type": "urn:ed-fi:api:security:authorization:namespace:access-denied:namespace-mismatch",
+                        "title": "Authorization Denied",
+                        "status": 403,
+                        "correlationId": null
+                    }
+                  """
+
+        @ignore
+        Scenario: 05 Post using an empty JSON body
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                    }
+                  """
+             Then it should respond with 400
+              And the response body is
+                  """
+                    {
+                      "detail": "Data validation failed. See 'validationErrors' for details.",
+                      "type": "urn:ed-fi:api:bad-request:data-validation-failed",
+                      "title": "Data Validation Failed",
+                      "status": 400,
+                      "correlationId": null,
+                      "validationErrors": {
+                        "$.namespace": [
+                          "namespace is required."
+                        ],
+                        "$.codeValue": [
+                          "codeValue is required."
+                        ],
+                        "$.shortDescription": [
+                          "shortDescription is required."
+                        ]
+                      },
+                      "errors": []
+                    }
+                  """
+
+        # Descriptors are not validating properly. DMS-295
+        @ignore
+        Scenario: 06 Ensure clients cannnot create a descriptor including soaces in the required attributes
```

Comment:
Suggested: "06 Ensure clients cannot create a descriptor only using spaces for the required attributes"

Reply:
Changed